### ISSUE TITLE
status_id variable is behaving correctly?

### ISSUE THREAD

**Issue by jrosen48** [jrosen48/comparing-sentiment#6]
I renamed `josh4liwc.rds` to `data_for_liwc.rds`

```{r}
> data_for_liwc %>% mutate(nchar_status_id = nchar(status_id)) %>% count(nchar_status_id)
# A tibble: 7 x 2
  nchar_status_id      n
            <int>  <int>
1               9      2
2              10     75
3              11    343
4              16     28
5              17    653
6              18 383774
7              19 196607
```

Just checking - what's up with the length of the `status_id` variable? not really a problem as we could also join on text, I think. And it may not be a problem in any case. I labeled this as a bug but it may not be.

**Comment by conradborchers** [comment]
Indeed, I have recognized this before. My best guess is either it has to do with attributes of the tweets or with how old the tweets are. In any way, this varying length has been apparent in other data sets I have seen as well.

**Comment by jrosen48** [comment]
got it - sounds good. okay if I keep this open for a bit to remind us to check this at some point? As you've addressed by using `.rds` files, if we use CSVs, sometimes these get treated as doubles by default, causing 0s to be erroneously removed. Shouldn't be an issue but would want to double-check.

**Comment by conradborchers** [comment]
Oh... Was not aware of this, I agree and will check later as well!

**Comment by jrosen48** [comment]
I think there is never a problem when we use `.rds` files - but when we save and the read `.csv` files (without specifying the column type), this can emerge!

**Comment by conradborchers** [comment]
Hi! This became an issue again now that I wanted to left join the combined LIWC results with the final raw data frame (including the state based hashtags). The number of unique status IDs for the raw data files and the LIWC data you gave me are not equal. The quickest solution might be to just re-run LIWC over the combined raw data files (NGSS and state-based hashtags), see the current branch "conrad2". Would you mind running LIWC one more time or looking at this again? Thanks...

**Comment by jrosen48** [comment]
Oye sorry. Lemme take the short/quick option for now. One moment.

**Comment by jrosen48** [comment]
just uploaded a new file (a compressed file as the size was causing some problems): `state-hashtags-liwc-results.csv.zip`

**Comment by jrosen48** [comment]
also pushed a small commit to the `conrad2` branch

**Comment by conradborchers** [comment]
Thanks! All seems to work.
